src/modules/aws_ecr/entity/repository_image.ts

```diff
@@ -62,6 +62,26 @@ export class RepositoryImage {
   })
   registryId?: string;
 
+  /**
+   * @public
+   * Date the image was pushed into the repository
+   */
+  @Column({
+    nullable: true,
+    type: 'timestamp with time zone',
+  })
+  pushedAt: Date;
+
+  /**
+   * @public
+   * Size of the image in MB
+   */
+  @Column({
+    nullable: true,
+    type: 'int',
+  })
+  sizeInMB: number;
```

Comment:
Not important for this PR, but maybe we should define a convention for units? (e.g. saving them always in MB)

Reply:
I believe it is MB everywhere